doc/VITAMIN.md

```diff
@@ -0,0 +1,66 @@
+## vitamin
+
+```JSON
+{
+  "id": "iron",
+  "type": "vitamin",
+  "vit_type": "vitamin",
+  "name": "Iron",
+  "excess": "hypervitaminosis",
+  "deficiency": "anemia",
+  "min": -12000,
+  "max": 3600,
+  "rate": "15 m",
+  "disease": [ [ -4800, -5600 ], [ -5601, -6400 ], [ -6401, -12000 ] ],
+  "disease_excess": [ [ 10, 19 ], [ 20, 29 ], [ 30, 40 ] ]
+},
+```
+### `id`
+Mandatory. The id of the vitamin.
+
+### `type`
+Mandatory. Must be `vitamin`.
+
+### `vit_type`
+The type of the vitamin. Valid values are:
+
+#### `vitamin`
+When simplified nutrition is enabled, this vitamin will not be added to any items and any time the game attempts to retrieve it from the player it will give 0.
+Only nutritional vitamins should have this type.
+
+#### `toxin`
+This is some toxic chemical or component. This currently has no effect.
+
+#### `drug`
+This is a drug. This currently has no effect.
+
+#### `counter`
+This is a counter for something, that is neither a toxin, vitamin, or drug. This currently has no effect.
+
+### `name`
+What the vitamin shows up as where vitamins are displayed, such as the vitamins display in the item menu.
+
+### `deficiency`
+The id of an effect that is triggered by a deficiency of this vitamin.
+
+### `excess`
+The id of an effect that is triggered by a deficiency of this vitamin.
+
+### `min`
+The smallest amount of this vitamin that the avatar can have.
```

Comment:
I guess "avatar" term could be confusing for modders and other people who don't follow the process of development.

Reply:
```suggestion
The smallest amount of this vitamin that the player can have.
```